### ISSUE TITLE
[CM] Subtitles stopped midloading screen do not go away until new subtitles overtake them

### ISSUE THREAD

**Issue by Zaphero1** [hzqst/MetaHookSv#263]
I tried setting a time duration for them but they still stay even with that. So like in this part in Echoes my closed captions for the vent breaking before loading and falling into a new area, 4 subtitles just stay onscreen all at once.

![20230313005000_1](https://user-images.githubusercontent.com/19298996/224618691-599b3efd-2170-4079-9c19-76326f9eae50.jpg)

**Comment by di57inct** [comment]
Possible duplicate of https://github.com/hzqst/MetaHookSv/issues/199

**Comment by hzqst** [comment]
try https://github.com/hzqst/MetaHookSv/releases/tag/v20230314a

**Comment by Zaphero1** [comment]
Oh dear it looks like that broke compatibility with MetaAudio; any subtitled sound doesn't play with it.

**Comment by Zaphero1** [comment]
Ah further info found on the previous build (linked below for clarity) tried staying in the same spot and waiting, and they eventually go away without new subtitles. I timed it and it took like 31 seconds, so if there is a regular autoclear that's working but it is taking a long time to do so.

https://github.com/hzqst/MetaHookSv/releases/tag/v20230313a

**Comment by Zaphero1** [comment]
AHHHH I think I figured out what the problem is. It's not that that captions from the last level are lasting into the next too long, it's that all the subtitles that show up in the first 30 seconds stay onscreen for that whole time and only cycle off from new subtitles in that time. I'm running into it when loading autosaves made at the start of levels. Let me try renaming the issue to match.

**Comment by Zaphero1** [comment]
I don't think so, it seems like if you load in a level with a captioned sound playing from second 0 that caption sometimes stays onscreen for many seconds longer than it should. It makes me think there might be a problem with how it records the initial time a sound plays right at the start of loading in or after a loading trigger.
